circle.yml

```diff
@@ -0,0 +1,11 @@
+machine:
+  node:
+    version: 6.5.0
+  environment:
+    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
```

Comment:
Is this explicit path necessary? Doesn’t Yarn automatically do this the same as NPM?